code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -36,6 +36,8 @@ const (
 	TagAddWriteMarker
 	TagAddBlock
 	TagAddOrOverwriteValidator
+	TagUpdateValidator
+	TagAddOrOverwriteReadMarker
```

Comment:
Unnecessary tag `TagAddOrOverwriteReadMarker` for this PR.

Reply:
I wonder how did it get there. Fixed in future commit.